code/datums/syndicate_buylist.dm

```diff
@@ -1136,56 +1107,57 @@ This is basically useless for anyone but miners.
 	desc = "Maybe paint a really insulting picture of your foe? To be honest, we have no idea what is even in these or where they came from, a huge crate of them just showed up at our warehouse around a month ago. We're sure it's something very handy, though!"
 	job = list("Chaplain")
 	vr_allowed = 0
-	blockedmode = list(/datum/game_mode/spy, /datum/game_mode/revolution)
+	can_buy = UPLINK_TRAITOR | UPLINK_SPY_THIEF
 
 /datum/syndicate_buylist/traitor/lawndarts
 	name = "Lawn Darts"
 	item = /obj/item/storage/box/lawndart_kit
 	cost = 0 // 20 brute damage, 10 bleed throwing weapon. Embed is nice but rad poison bow is stealthier and more effective
 	desc = "Three deadly throwing darts that embed themselves into your target."
 	job = list("Assistant","Technical Assistant","Medical Assistant","Staff Assistant","Bartender","Clown")
-	blockedmode = list(/datum/game_mode/revolution)
+	can_buy = UPLINK_TRAITOR | UPLINK_SPY_THIEF | UPLINK_SPY
 
 // round specific
 
-/datum/syndicate_buylist/generic/revflash
+/datum/syndicate_buylist/generic/head_rev
+	name = "Head Rev Buylist Parent"
```

Comment:
then make it abstract

Reply:
fixed for all cases in the buylist datums